packages/react/src/components/Pivot/Pivot.styles.ts

```diff
@@ -145,7 +145,7 @@ const getLinkStyles = (
                   backgroundColor: semanticColors.primaryButtonBackgroundHovered,
                   color: semanticColors.primaryButtonText,
                 },
-                '&:active': {
+                ':active': {
```

Comment:
This was a semi-unrelated fix -- using `&` doesn't work inside nested blocks, and is unnecessary in this selector